docs/data/migration/migration-tree-view-v7/migration-tree-view-v7.md

```diff
@@ -0,0 +1,178 @@
+---
+productId: x-tree-view
+---
+
+# Migration from v7 to v8
+
+<p class="description">This guide describes the changes needed to migrate the Tree View from v7 to v8.</p>
+
+## Introduction
+
+This is a reference guide for upgrading `@mui/x-tree-view` from v7 to v8.
+
+## Start using the new release
+
+In `package.json`, change the version of the Tree View package to `^8.0.0`.
+
+```diff
+-"@mui/x-tree-view": "7.x.x",
++"@mui/x-tree-view": "^8.0.0",
```

Comment:
Shouldn't this next in the pre-release phase like we did in earlier releases?
```tsx
"@mui/x-tree-view": "next",
```
CC @flaviendelangle

Reply:
In the migration guide we used the exact version, not `next`: https://mui.com/x/migration/migration-data-grid-v6/#start-using-the-new-release

No strong opinion here though

Reply:
IIRC we used to update to the exact version on stable release.
A reference original commit on alpha for the Data Grid migration guide v6->v7: https://github.com/mui/mui-x/pull/10846/files#diff-32b8ccbf3738b9aa1a6038454e185a6aea9b2c175ab3ab7f1eb2302d8b625ff2

Reply:
OK, that explains a lot :laughing: 
I'll update the tree view and pickers migration guides then :+1:

Reply:
#15091